unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/States/EditorMode/BiwGodMode.cs

```diff
@@ -124,8 +119,8 @@ public override void Dispose()
 
         gizmoManager.OnChangeTransformValue -= EntitiesTransfromByGizmos;
 
-        GameObject.Destroy(lookAtT.gameObject);
-        GameObject.Destroy(builderGO);
+        if (lookAtT.gameObject != null)
```

Comment:
```suggestion
        if (lookAtTransform.gameObject != null)
```
Please don't abbreviate if it can be avoided